src/layers/XRLayer/xr-layer-fragment.js

```diff
@@ -0,0 +1,27 @@
+export default `#version 300 es
+#define SHADER_NAME xr-layer-fragment-shader
+precision highp float;
+precision highp int;
+precision highp usampler2D;
+
+
+// our texture
+uniform usampler2D redTexture;
+uniform usampler2D greenTexture;
+uniform usampler2D blueTexture;
+// range
+uniform uint redSliderValue;
+uniform uint blueSliderValue;
+uniform uint greenSliderValue;
```

Comment:
These are set to default values for now - that will change when I bring in the sliders.  I didn't want to put too much into this PR though.